build/utils.py

```diff
@@ -21,6 +21,24 @@
 # Third Party
 import torch
 from accelerate.commands.launch import launch_command_parser
+import shutil
+
+
+def copytree(source, destination):
+    if not os.path.exists(destination):
+        os.makedirs(destination)
+        shutil.copystat(source, destination)
+    file_list = os.listdir(source)
+    # Have a list of directory objects, now iterate over them.
+    for item in file_list:
+        source_file = os.path.join(source, item)
+        destination_file = os.path.join(destination, item)
+        if os.path.isdir(source_file):
+            # recursive call for subdirectories
+            copytree(source_file, destination_file)
+        else:
+            # straight copy.
+            shutil.copy2(source_file, destination_file)
```

Comment:
What about symlinks? Do we want to set `follow_symlinks=True`? If a symlink (for some reason) is in the source directory, would this method fail or would the symlink just not get copied over?

Reply:
I pulled the symlink functionality. Do we need it added back?

Reply:
Symlinks are probably not needed as the checkpoints don't contain symlinks. But I'm just curious if setting `follow_symlinks=True` does copy over symlinks. In addition, if this method is just for copying the checkpoint, then I would rename the method to `copy_checkpoint()` instead of copytree to avoid confusion